source/_integrations/meteoalarm.markdown

```diff
@@ -34,7 +34,7 @@ name:
   default: meteoalarm
   type: string
 country:
-  description: The 2 letters of your country
+  description: The full name of your country in english format
```

Comment:
```suggestion
  description: The full name of your country in English format
```

Reply:
It should be in lowercases, see: https://feeds.meteoalarm.org/feeds/meteoalarm-legacy-atom-netherlands